.gitignore

```diff
@@ -2,7 +2,6 @@
 .tox/
 __pycache__/
 .ipynb_checkpoints/
-tests/output/
```

Comment:
why is this removed?

Reply:
It wasn't removed. It was just put beside an existing `test/input/` line just for logical grouping. I could leave that out of the PR if you want too but it is very small.